monitoring/src/check-page/tcfv2.ts

```diff
@@ -95,12 +77,12 @@ const checkSubsequentPage = async (
 	// There is no CMP since this we have already accepted this on a previous page.
 		await checkTopAdHasLoaded(page);
 	await Promise.all([
-		clearCookies(await page.target().createCDPSession()),
+			clearCookies(page),
 		clearLocalStorage(page)
 	]);
 	await reloadPage(page);
 	await checkTopAdDidNotLoad(page);
-	await clickAcceptAllCookies(config, page);
+		await clickAcceptAllCookies(config, page, `Yes I'm Happy`);
```

Comment:
The reason we moved away from using the button text to identify and click on a button was because this often changes as part of a cookie banner test.  Then we get intermittent failures which are tricky to identify.

Reply:
This is only for logging, same as it is at the moment. I only moved the function to a central place as it was duplicated, no functional changes.